### PR TITLE
Ansible summary script enhancements

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

File: ansible/roles/kstest-master/files/scripts/kstests_history.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 parser.add_argument("--isomd5sum_filename", "-m", metavar="ISO_MD5_SUM_FILENAME", type=str,
                     default="isomd5sum.txt", help="Name of the file with md5 sum of boot iso")
 parser.add_argument("--status_count", "-s", metavar="NUMBER_OF_LATEST_RESULTS", type=int,
-                    default=3, help="Number of latest results to be used for status")
+                    default=1, help="Number of latest results to be used for status")
```
Comment (jkonecny12) [inline ansible/roles/kstest-master/files/scripts/kstests_history.py]: What's the reason for this change? I think it's better to have 3 runs by default than only one?

File: ansible/roles/kstest-master/files/scripts/kstests_history.py [hunk 2]
```diff
@@ -25,6 +25,9 @@
 md5sum_filename = args.isomd5sum_filename
```
Comment (jkonecny12) [inline ansible/roles/kstest-master/files/scripts/kstests_history.py]: Typo in commit b71a0283d2c954ebe52e44fcec2009434b6004f1 :
```
summary html: add test run time to header *fo* summary html page
```

File: ansible/roles/kstest-master/files/scripts/kstests_history.py [hunk 3]
```diff
@@ -53,13 +53,13 @@ def get_anconda_ver(result_path):
...
 old_isomd5 = ""
 for result_dir in sorted(os.listdir(results_path)):
-    count = count + 1
     result_path = os.path.join(results_path, result_dir)
 
     report_file = os.path.join(result_path, report_filename)
     if not os.path.exists(report_file):
         print("No {} found, skipping".format(report_file), file=sys.stderr)
         continue
+    count = count + 1
```
Comment (jkonecny12) [inline ansible/roles/kstest-master/files/scripts/kstests_history.py]: Please move this one line below. It's easier to confuse it with the if above without an empty space.

I mean change this:
```
    continue
count = count + 1

anaconda_ver = ""

with open....
```

to:

```
    continue

count = count + 1
anaconda_ver = ""

with open...
```

File: ansible/roles/kstest-master/files/scripts/kstests_history.py (hunk 1 above)
Comment (rvykydal) [reply ansible/roles/kstest-master/files/scripts/kstests_history.py]: I don't remember, I've removed the commit from the PR.

File: ansible/roles/kstest-master/files/scripts/kstests_history.py (hunk 2 above)
Comment (rvykydal) [reply ansible/roles/kstest-master/files/scripts/kstests_history.py]: Fixed

File: ansible/roles/kstest-master/files/scripts/kstests_history.py (hunk 3 above)
Comment (rvykydal) [reply ansible/roles/kstest-master/files/scripts/kstests_history.py]: Fixed
